apps/rush-lib/src/cli/actions/VersionAction.ts

```diff
@@ -239,7 +239,7 @@ export class VersionAction extends BaseRushAction {
     });
 
     if (packageJsonUpdated) {
-      git.addChanges('.', this.rushConfiguration.versionPolicyConfigurationFilePath);
+      git.addChanges(this.rushConfiguration.versionPolicyConfigurationFilePath, '.');
```

Comment:
This is the default, so you can just cut the last param off here

Reply:
Removed the last param 👍